src/Blueprint/Blueprint/Blueprint.format.ps1xml

```diff
@@ -183,6 +183,22 @@
                 <Label>LockMode</Label>
                   <ScriptBlock>$_.Locks.Mode.ToString()</ScriptBlock>
               </ListItem>
+				<ListItem>
+					<Label>ExcludedActions</Label>
+					<ScriptBlock>
+						If (-not ([string]::IsNullOrEmpty($_.Locks.ExcludedActions))){
+						($_.Locks.ExcludedActions) -join "; "
```

Comment:
@filiz, what delimiter do we usually use for joining list items?

Reply:
It depends on the context. For this one, we considered comma, semi colon and line item and ended up using semi-colon. It'd be good to have your thoughts on that. @Xynoclafe could you send an email to Lei, Jorge and me showing how those 3 options would look in the output? We can update it if we thought some other delimiter looks better.

---
In reply to: [413184933](https://github.com/Azure/azure-powershell/pull/11661#discussion_r413184933) [](ancestors = 413184933)

Reply:
Okay